File: nebula2/code/contrib/npythonserver/bin/stdlight.py
#-------------------------------------------------------------------------------
#   Defines the standard lighting for npyviewer.exe
#-------------------------------------------------------------------------------

oldCwd = psel()
sel('/usr/scene')

stdLight = new('nspotlightnode', 'stdlight')
stdLight.setposition(1000, 500, 1000)
stdLight.setorthogonal(10, 10, 0.25, 175)
stdLight.setvector('LightDiffuse',   1,   1,   1,   1)
stdLight.setvector('LightSpecular',  1,   1,   1,   1)
stdLight.setvector('LightAmbient',   1,   1,   1,   1)
stdLight.setvector('LightDiffuse1',  0.3, 0.3, 0.3, 1.0)
stdLight.setvector('LightSpecular1', 0.3, 0.3, 0.3, 1.0)
stdLight.settexture('LightModMap', 'textures:system/spotlight.dds')

sel(oldCwd)

File: nebula2/code/contrib/nspatialdb/bin/stdlight.lua
-------------------------------------------------------------------------------
--   Defines the standard lighting for nviewer.exe
-------------------------------------------------------------------------------

pushcwd('.')

sel('/usr/scene')

new('ntransformnode','stdlight')
    sel('stdlight')
    new('nspotlightnode', 'l0')
	l = lookup('l0')
	sel('l0')
	l:setposition(0,10,0)
	l:seteuler(-30,45,0)
	l:setorthogonal(1000,1000,0.25,175)
	l:setvector('LightDiffuse',0.8,0.8,0.8,1)
	l:setvector('LightSpecular',1,1,1,1)
	l:setvector('LightAmbient',0.2,0.2,0.2,1)
	l:settexture('LightModMap','textures:system/white.dds')
    sel('..') 
sel('..') 

popcwd()




